server/server.js

```diff
@@ -0,0 +1,57 @@
+import express from "express";
+import axios from "axios";
+import dotenv from "dotenv";
+import cors from "cors";
+
+dotenv.config();
+
+const app = express();
+const PORT = 8080;
+
+app.use(
+  cors({
+    origin: process.env.URL || "http://localhost:5173",
+    allowedHeaders: "Origin, X-Requested-With, Content-Type, Accept",
+  })
+);
+
+app.get("/weather", async (req, res) => {
+  try {
+    const apiKey = process.env.API_KEY;
```

Comment:
<img width="1406" alt="image" src="https://github.com/bpb2008/WeatherApp/assets/152167/b71e7cee-24e6-4cef-a3cd-d4322f7611f0">
When you submit a PR to a senior dev for review, don't forget to mention that they need to configure their own .env files. When I ran your code it wouldn't work because I didn't first get an API key or put it in a .env file. Be sure to mention this in some setup instructions in your README. (Of course I know what I must do to set it up without your instructions but for people unfamiliar with what you're doing or anyone else in the company you're at for that matter, documenting the setup clearly in the README is a must and expected - as well as for interviews if you do a take-home demo app test.)